README.Rmd

```diff
@@ -178,4 +178,4 @@ Thanks to:
 
 ## Code of Conduct
 
-Please note that the readr project is released with a [Contributor Code of Conduct](https://readr.tidyverse.org/CONDUCT.html). By contributing to this project, you agree to abide by its terms.
```

Comment:
FWIW we've decided to no longer go out of our way to place this text in README, since GitHub features a code of conduct rather prominently, as do our pkgdown sites.

But we also aren't too worried about removing this text. It's a weak recommendation.

So we can remove this section. In which case you'll need to do `devtools::build_readme()` again.